todo_app/data/trello_items.py

```diff
@@ -0,0 +1,51 @@
+import requests
+import os
+
+from todo_app.item import Item
+
+auth = {

```

Comment:
Nothing wrong with this, but be warned it might catch you out slightly in the next exercise if you care about what these are set to for your tests; since this is just at the top level of the file (i.e. not in a function/class), it executes when the file is imported, which can mean your tests don't have time to override the environment variables with test values. Possibly won't be an issue, but if you want to avoid it we could make this a function:
```
def get_auth():
    return {
        ...
```